Readme.md

```diff
@@ -5,7 +5,7 @@ This project it's to learn DevOps
 1. [Python exercises](python)
     - [Hello world](python/hello-world.py)
 2. [Git commands](git_commands.md)
-3. [Commands](commands.md)
+3. [Commands](commands..imd)
```

Comment:
Kenny, please check your suggestion. I believe you have a typo as "commands..imd" is not a valid file on my repository.